### ISSUE TITLE
Android emulator does not open until CTRL+C is pressed during "Starting emulator..."

### ISSUE THREAD

**Issue by rmartin16** [beeware/briefcase#1573]
### Describe the bug

Several users in Discord have reported `briefcase run android` results in Briefcase getting stuck at the Wait Bar for `Starting emulator...`. When CTRL+C is pressed, Briefcase exits and the emulator opens.

This issue can be worked around by leaving the emulator open and running `briefcase run android` again and then choosing the running emulator in the list

### Steps to reproduce

Not sure yet...

### Expected behavior

Emulator opens while Briefcase is running.

### Environment

- Operating System: all reports have been Windows 10 22H2 so far
- Python version: 3.10, 3.11
- Software versions:
  - Briefcase: 0.3.16

### Logs

[briefcase.2023_12_06-23_21_25.run(1).log](https://github.com/beeware/briefcase/files/13666570/briefcase.2023_12_06-23_21_25.run.1.log)
[briefcase.2023_12_13-21_53_23.run(1).log](https://github.com/beeware/briefcase/files/13666572/briefcase.2023_12_13-21_53_23.run.1.log)
[briefcase.2023_12_04-23_10_36.run.log](https://github.com/beeware/briefcase/files/13666574/briefcase.2023_12_04-23_10_36.run.log)

**Comment by JaxkDev** [comment]
Can add Python 3.8 + 23H2 to the list, seems to replicate 100% of the time...

**Comment by f-bdolan** [comment]
Additionally, when it does start, my application is not installed

**Comment by rmartin16** [comment]
> Additionally, when it does start, my application is not installed

@f-bdolan: correct; when you press CTRL+C and the emulator opens, your app will not have been installed. However, if you run `briefcase run android` again and select the "running emulator", Briefcase will install your app then.

**Comment by f-bdolan** [comment]
Ah! Excellent, so there is a work-around.  For those of you just showing up:

If `briefcase run android` stalls on "starting":
1. Run `briefcase run android`
2. Pick an emulator (if you use beePhone it will have debugging options turned on)
3. Wait a moment to stall
4. Hit Ctrl-C
5. Leave the emulator open
6. Run `briefcase run android` again and choose the SAME emulator
7. It will install
